src/backend/access/appendonly/appendonlywriter.c

```diff
@@ -923,30 +923,34 @@ usedByConcurrentTransaction(AOSegfileStatus *segfilestat)
  */
 int
 addCandidateSegno(AOSegfileStatus **maxSegno4Seg, int segment_num, AOSegfileStatus *candidate_status, bool isKeepHash)
```

Comment:
Because github default ts=8, which may a lot indentation looks strange. 
However our code convention is to set ts=4.
e.g. When you review code, please append '?ts=4' to web URL as below:
https://github.com/apache/incubator-hawq/pull/1187/files?ts=4